notebooks/mg-20230513-00-embedding-ideas.ipynb

```diff
@@ -0,0 +1,768 @@
+{
```

Comment:
<p>How long did this take to run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>about 10 mins</p>